.github/ISSUE_TEMPLATE/2.bug.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Bug report 🐛
+about: Create a bug report for Material-UI.
+---
+
+<!-- Provide a general summary of the issue in the Title above -->
+
+<!--
+  Thank you very much for contributing to Material-UI by creating an issue! ❤️
+  To avoid duplicate issues we ask you to check off the following list.
+-->
+
+<!-- Checked checkbox should look like this: [x] -->
+
+- [ ] This is not a v0.x issue. <!-- (v0.x is no longer maintained) -->
```

Comment:
Should we change this to include other versions?

Reply:
Like, this is a v4.x issue?

Reply:
Yeah, was only thinking the same thing the other day. 

How about:

```suggestion
- [ ] The issue is present in the latest release. -->
```